tests/test_layout.py

```diff
@@ -14,10 +14,14 @@ class Box(BaseBox):
         pass
 
     def layout(self, root, viewport):
-        # A simple layout scheme that allocats twice the viewport size.
+        # A simple layout scheme that allocates twice the viewport size,
+        # and has a minimum content size of half the viewport size
         root.layout.content_width = viewport.width * 2
         root.layout.content_height = viewport.height * 2
 
+        root.layout.min_content_width = viewport.width // 2
+        root.layout.min_content_height = viewport.height // 2
```

Comment:
I don't think this code is actually being used.

```suggestion
        pass
```

Reply:
More than that - the entire `layout()` method seems completely unnecessary. There's nothing using that method, AFAICT.